config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -304,6 +307,38 @@ public void addJaxProviders() {
         addSimpleComponent(XPathFactoryProvider.class);
     }
 
+    public void addIdentity() {
```

Comment:
Please refactor, this method is too long. Most of this code seems irrelevant to ContainerCluster. If I read correctly, it just sets two properties on the cluster, which could perhaps be set from the builder. I'm not sure, but let's look at this again after this method has been refactored to become more legible.